examples/01-mathematical-operations/matrix-operations.py

```diff
@@ -0,0 +1,95 @@
+"""
+.. _ref_matrix-operations:
+
+Matrix Operations
+~~~~~~~~~~~~~~~~~
+
+This example shows how to do some matrix operations, including basic mathematical operation (power, add and multiply by
+a constant, add field containers and invert ) and separating and assembling fields and fields containers
+
+"""
+
+###############################################################################
+# Import the ``ansys.dpf.core`` module, included examples file, and the ``DpfPlotter``
+# module.
+from ansys.dpf import core as dpf
+from ansys.dpf.core import examples
+import ansys.dpf.core.operators.math as maths
+
+###############################################################################
+# Open an example and print the ``Model`` object. Here a result file from a crankshaft
+# under load simulation is used.
+# The :class:`Model <ansys.dpf.core.model.Model>` class helps to organize access
+# methods for the result by keeping track of the operators and data sources
+# used by the result file.
+#
+# Printing the model displays this metadata:
+#
+# - Analysis type
+# - Available results
+# - Size of the mesh
+# - Number of results
+#
+my_model = dpf.Model(examples.find_complex_rst())
+my_mesh = my_model.metadata.meshed_region
+# print(my_model)
+###############################################################################
+# Get the stress tensor and define it's scoping. Here, only three nodes will be take into account to facilitate the
+# results visualisation
+my_nodes_scoping = dpf.Scoping(ids=[38, 37, 36], location=dpf.locations.elemental)
+my_stress = my_model.results.stress(mesh_scoping=my_nodes_scoping).eval()
+
+# Here we need to average the result from 'elemental_nodal' to an 'elemental' location to
+# facilitate the visualisation of the plot
+my_avg_stress = dpf.operators.averaging.to_elemental_fc(fields_container=my_stress, mesh=my_mesh).eval()
```

Comment:
```suggestion
my_avg_stress = my_model.results.stress(mesh_scoping=my_nodes_scoping).on_location(dpf.locations.elemental).eval()
```
You can also directly request the result as "elemental" like shown above